doc/release-notes.md

```diff
@@ -116,7 +116,8 @@ Configuration option changes
   defaults to being off, so that changes in policy and disconnect/ban behavior
   will not cause a node that is whitelisting another to be dropped by peers.
   Users can still explicitly enable this behavior with the command line option
-  (and may want to consider letting the Bitcoin Core project know about their
+  (and may want to consider [contacting](https://bitcoincore.org/en/contact/)
```

Comment:
Shouldn't this link to the public issue tracker?

Reply:
I didn't like that the previous text left it unspecified how to contact the project (and, since release notes are posted to the protocol dev mailing list, I was worried people would reply there, which seemed off-topic to me).  I chose the website because it lists the following item that seemed like the most appropriate way for individuals and businesses to inform the project about their use of a feature:

> For general enquiries and press: contact@bitcoincore.org (not for support).

I think people would be more reluctant to open an "issue" for something that isn't broken (they're using the feature as designed) and which may require business users to publicly describe how they use the software in a toolchain that may produce normally-unrelayable transactions.

However, I don't know who reads the above email or whether they actually want to receive reports about who uses this feature, so I'm happy to switch to the issue tracker if that's preferred.

CC: @sdaftuar (who sought the contact from users of the `whitelistforcerelay` option)

Reply:
Thanks for the reply. I think that makes sense and the contact page also links to the GitHub, so they could choose either email or GitHub issue.